### PR TITLE
Log that PgBouncer is enabled.

### PR DESCRIPTION
**Pull request by do4gr**
Fixes https://github.com/prisma/prisma-engines/issues/877

### CODE REVIEW

File: src/connector/postgres.rs [hunk 1]
```diff
@@ -424,6 +424,17 @@ impl PostgreSql {
             }
         }));
 
+        if url.query_params.pg_bouncer {
+            #[cfg(not(feature = "tracing-log"))]
+            {
+                info!("PgBouncer is enabled.");
```
Comment (janpio) [inline src/connector/postgres.rs:430]:
```suggestion
                info!("PgBouncer mode is enabled.");
```

File: src/connector/postgres.rs [hunk 2]
```diff
@@ -424,6 +424,17 @@ impl PostgreSql {
...
+            }
+            #[cfg(feature = "tracing-log")]
+            {
+                tracing::info!("PgBouncer is enabled.");
```
Comment (janpio) [inline src/connector/postgres.rs:434]:
```suggestion
                tracing::info!("PgBouncer mode is enabled.");
```

File: src/pooled.rs [hunk 3]
```diff
@@ -301,6 +285,24 @@ impl Builder {
             connect_timeout: self.connect_timeout,
         }
     }
+
+    fn log_start(info: &ConnectionInfo, connection_limit: usize) {
+        let family = info.sql_family();
+        #[cfg(not(feature = "tracing-log"))]
+        {
+            if info.pg_bouncer() {
+                info!("PgBouncer mode is enabled.");
```
Comment (pimeys) [inline src/pooled.rs:294]: Could this even be in the same line that tells pool is started?

File: src/single.rs [hunk 4]
```diff
@@ -133,26 +132,33 @@ impl Quaint {
...
-    }
-
-    fn log_start(family: SqlFamily, connection_limit: u32) {
+    fn log_start(info: &ConnectionInfo) {
+        let family = info.sql_family();
         #[cfg(not(feature = "tracing-log"))]
         {
-            info!("Starting a {} pool with {} connections.", family, connection_limit);
+            if info.pg_bouncer() {
+                info!("PgBouncer mode is enabled.");
```
Comment (pimeys) [inline src/single.rs:145]: Here too. Put it to the same line that tells connection is started.

File: src/connector/connection_info.rs [hunk 5]
```diff
@@ -192,6 +192,20 @@ impl ConnectionInfo {
...
+    pub fn pg_bouncer(&self) -> bool {
+        match self {
+            #[cfg(feature = "postgresql")]
+            ConnectionInfo::Postgres(url) => url.pg_bouncer(),
+            #[cfg(feature = "mysql")]
+            ConnectionInfo::Mysql(_) => false,
+            #[cfg(feature = "mssql")]
+            ConnectionInfo::Mssql(_) => false,
+            #[cfg(feature = "sqlite")]
+            ConnectionInfo::Sqlite { .. } => false,
```
Comment (mavilein) [inline src/connector/connection_info.rs:205]: shouldn't the last 3 cases just be `_ => false`? Would be easier to process visually imo.

Comment (do4gr) [reply src/connector/connection_info.rs:205]: True

**Pull request merged**